project/SparkBuild.scala

```diff
@@ -418,6 +420,8 @@ object SparkBuild extends PomBuild {
 
   enable(HiveThriftServer.settings)(hiveThriftServer)
 
+  enable(SparkConnect.settings)(pipelinesCore)
```

Comment:
will the setting be exactly the same as `SparkConnect.settings`?

Reply:
I think these will actually drift in the future. For now, I removed the enable call and will just rely on the shared settings